modules/table/src/main/java/org/apache/ignite/internal/table/distributed/StorageUpdateHandler.java

```diff
@@ -291,12 +291,12 @@ public void handleTransactionCleanup(UUID txId, boolean commit,
      * @param commitTimestamp Commit timestamp. Not {@code null} if {@code commit} is {@code true}.
      * @param onApplication On application callback.
      */
-    private void handleTransactionCleanup(Set<RowId> pendingRowIds, boolean commit,
+    private void handleTransactionCleanup(UUID txId, Set<RowId> pendingRowIds, boolean commit,
             @Nullable HybridTimestamp commitTimestamp, Runnable onApplication) {
         if (commit) {
-            handleTransactionCommit(pendingRowIds, commitTimestamp, onApplication);
+            handleTransactionCommit(txId, pendingRowIds, commitTimestamp, onApplication);
         } else {
-            handleTransactionAbortion(pendingRowIds, onApplication);
+            handleTransactionAbortion(txId, pendingRowIds, onApplication);
         }
```

Comment:
I'd rather add the check for `pendingRowIds` emptiness here, in order to not call `storage.runConsistently` if we have nothing to cleanup.